homeassistant/components/zha/core/registries.py

```diff
@@ -115,7 +115,17 @@ def get_deconz_radio():
         zha.DeviceType.LEVEL_CONTROLLABLE_OUTPUT: LIGHT,
         zha.DeviceType.ON_OFF_LIGHT: LIGHT,
         zha.DeviceType.DIMMABLE_LIGHT: LIGHT,
-        zha.DeviceType.COLOR_DIMMABLE_LIGHT: LIGHT
+        zha.DeviceType.COLOR_DIMMABLE_LIGHT: LIGHT,
+        zha.DeviceType.ON_OFF_LIGHT_SWITCH: BINARY_SENSOR,
```

Comment:
Remove this as we no longer use binary sensor for remotes.

Reply:
Fixed these instance by matching them to SWITCH and LIGHTs since they are target devices according to the Zigbee documentation.